exercises/react/example.rs

```diff
@@ -1,12 +1,13 @@
 use std::collections::HashMap;
 
-pub type CellID = usize;
+pub type InputCellID = usize;
+pub type ComputeCellID = usize;
```

Comment:
I note that these are typedefs, not newtypes. As such, the compiler treats `InputCellID`, `ComputeCellID`, and `usize` interchangeably. If we want it to enforce real type safety for us, we'll need to use newtypes:

```rust
pub struct InputCellID(usize);
pub struct ComputeCellID(usize);
// the above two types _cannot_ be interchanged
```